content/docs/guide-development.md

```diff
@@ -117,6 +117,31 @@ https://renkulab.io/
 
 TODO: explain how to run server
 
+### Developing plugins for Renku
+
+Renku provides a number of hooks that can be used to add additional customized metadata. It provides also the possibility to add CLI commands.
+
+[https://renku.readthedocs.io/en/stable/renku-python/docs/reference/plugins.html](https://renku.readthedocs.io/en/stable/renku-python/docs/reference/plugins.html)
+
+Two plugins have been developed
```

Comment:
the sentence does not end. The next two headers start paragraphs, they do not continue sentence.